sender-app/configuration/application.properties

```diff
@@ -195,3 +200,18 @@ pgp.sender.password=
 # User name of the PGP receiver key pair
 pgp.sender.receiverUserId=
 # ----------------------------------------------------------------------------------------------------------------------
+
```

Comment:
I think these oauth properties already  exist in this file

Reply:
in https://raw.githubusercontent.com/mekomsolutions/openmrs-dbsync/master/sender-app/configuraton/application.properties ? Not sure...

Reply:
They are receiver props and not sender

Reply:
they are required in the sender app as you mentionned in a previous message. Try to launch the sender without them and it will fail

Reply:
Ah! That's correct